lam4-backend/src/Lam4/Expr/Printer.hs

```diff
@@ -135,6 +135,21 @@ instance Print Int where
 instance Print Double where
   prt _ x = doc (shows x)
 
+
+-- Function arguments are separated by newlines
+newtype FunArg = FunArg {unFunArg :: String}
+
+mkFunArg :: T.Text -> FunArg
+mkFunArg t = FunArg [I.i|#{t} : TypeMissing|]
```

Comment:
I added this, because I couldn't find the information about the argument types in the CST. Here's an example, when printing the file [examples/infix_pred_app.l4](https://github.com/smucclaw/lam4/blob/main/examples/infix_pred_app.l4).

* Original:
    ```
    GIVEN (some_person: Person 
           another_person: Person)
    DECIDE some_person `has helped` another_person
    IF True
    ```

* CST:
    ```
    Rec
    "has helped"
    _2
    (Predicate
       RuleMetadata
         { originalRuleRef = Nothing
         , transparency = Transparent
         , description = Nothing
         }
       [ "some_person" _3 , "another_person" _4 ]
       (Lit (BoolLit True)))
    ```
And this is how it's prettyprinted currently.

```
GIVEN (some_person : TypeMissing
       another_person : TypeMissing)
DECIDE `has helped`
IF True
```

(The CST is missing also the information about how the arguments are used in the body, but I will talk about that in another comment.)